### PR TITLE
updating a few of the help texts

### PR DESCRIPTION
**Pull request by acornett21**
- Updating a few of the help cmd text values since a few partners have had confusion around these values.

Below is the new output
```
Flags:
      --certification-project-id string   Certification Project ID from connect.redhat.com/projects/{certification-project-id}/overview
                                          URL paramater. This value may differ from the PID on the overview page. (env: PFLT_CERTIFICATION_PROJECT_ID)
  -h, --help                              help for container
      --pyxis-api-token string            API token for Pyxis authentication (env: PFLT_PYXIS_API_TOKEN)
      --pyxis-env string                  Env to use for Pyxis submissions. (default "prod")
      --pyxis-host string                 Host to use for Pyxis submissions. This will override Pyxis Env. Only set this if you know what you are doing.
                                          If you do set it, it should include just the host, and the URI path. (env: PFLT_PYXIS_HOST)
  -s, --submit                            submit check container results to red hat

Global Flags:
  -d, --docker-config string   Path to docker config.json file. This value is optional for publicly accessible images.
                               However, it is strongly encouraged for public Docker Hub images,
                               due to the rate limit imposed for unauthenticated requests. (env: PFLT_DOCKERCONFIG)
  -l, --list-checks            lists all the checks run for a given check

```

Signed-off-by: Adam D. Cornett <adc@redhat.com>

### CODE REVIEW

File: cmd/check_container.go
Comment (bcrochet) [inline cmd/check_container.go:252]: Can you format this like the other? But with the newlines at the end? Convince me why newlines would go at the beginning?

File: cmd/check_container.go
Comment (bcrochet) [inline cmd/check_container.go:259]: Same with format.

**Review by bcrochet — CHANGES_REQUESTED** [review]
Couple of comments about formatting.

File: cmd/check_container.go
Comment (acornett21) [reply cmd/check_container.go]: Done.

File: cmd/check_container.go
Comment (acornett21) [reply cmd/check_container.go]: Done.

File: cmd/check_container.go
Comment (komish) [inline cmd/check_container.go:258]: Is there really no other place to get this value than from the URL?

If provided the PID from the overview page, could we instead look up the ospid?

**Review by komish — COMMENTED** [review]
Few comments

File: cmd/check_container.go
Comment (acornett21) [reply cmd/check_container.go:258]: @komish For current projects, the URL value _is_ the PID value, but for legacy projects they do not match and partners give us the value in the PID field which is also the ospid value for those legacy projects. As far as I know all pyxis api's take the URL value, so not sure we can cross reference, nor would I really be in favor for another call to pyxis.

File: cmd/check_container.go
Comment (komish) [reply cmd/check_container.go]: Four line breaks just seem like a lot. Totally a nit. Feel free to disregard.

--

As it pertains to adding an example value: 

If we're really concerned about partners using this when they shouldn't (as in, when they mean to use the other flag), then perhaps we need to massage how we indicate to the user that this is unlikely to be the flag they want. Our current verbiage (`Only set this if you know what you are doing.) doesn't really help in this regard. 

Not including an example value to prevent folks from using it when they shouldn't doesn't make sense.

If we really want to go that route, just mark the flag hidden. I'd prefer not to do this.

Comment (acornett21) [reply cmd/check_container.go]: I didn't change any of the verbiage here, or come up with it, it's existing. I just changed the formatting so the code was readable and not 800 characters of scroll city. For this flag I'd be fine with it being hidden, since it would be for us as developers to use to test potential issues between the internal/external instances or say a pyxis route change.

Comment (komish) [reply cmd/check_container.go]: I'll let @bcrochet chime in. I'll toss in the LGTM if Brad's good with it.

**Review by komish — APPROVED** [review]
/lgtm

**Review by jomkz — APPROVED** [review]

**Pull request merged**
